.golangci.yaml

```diff
@@ -0,0 +1,15 @@
+run:
+  timeout: 5m
+
+  skip-dirs:
+    - internal
+    - pkg/registry
+
+linters:
```

Comment:
We're running:
```
linters:
  enable:
  - asciicheck
  - deadcode
  - depguard
  - errcheck
  - errorlint
  - gofmt
  - goimports
  - gosec
  - gocritic
  - importas
  - prealloc
  - revive
  - misspell
  - stylecheck
  - tparallel
  - unconvert
  - unparam
  - whitespace
```

Reply:
```
$ golangci-lint run | grep pkg | wc -l
      58
```

We'll get there. Thanks for sharing this!

Reply:
57 of those are probably errcheck on `io.Copy` in `pkg/registry` 😉

Reply:
```
skip-dirs:
    - internal
    - pkg/registry
```

😎